code/modules/mob/living/simple_animal/hostile/hivebots/hivebot_projectiles.dm

```diff
@@ -1,11 +1,11 @@
 /obj/item/projectile/bullet/pistol/hivebotspike
 	name = "spike"
 	damage = 10
-	armor_penetration = 40
+	armor_penetration = 20
```

Comment:
Makes this ammo useless against armour, no.

Reply:
changed to 30ap